frontend/tests/jest/tests/ProductSection.test.tsx

```diff
@@ -4,112 +4,183 @@ import {
+         (expect(warning) as any).toBeInTheDocument();
+         (expect(warning.textContent) as any).toContain('Mocked Warning');
       });
 
-      renderWithAppContext(
-         <ProductSection
-            product={getMockProduct()}
-            selectedVariant={productVariant}
-            onVariantChange={jest.fn()}
-            internationalBuyBox={null}
-         />
-      );
-
-      const warning = await screen.findByTestId('product-warning');
-      (expect(warning) as any).toBeInTheDocument();
-      (expect(warning.textContent) as any).toContain('Mocked Warning');
+      test('descriptors do not render', async () => {
+         const productVariant = getMockProductVariant({
+            note: '',
+            warning: '',
+            disclaimer: '',
+         });
+
+         renderWithAppContext(
+            <ProductSection
+               product={getMockProduct()}
+               selectedVariant={productVariant}
+               onVariantChange={jest.fn()}
+               internationalBuyBox={null}
+            />
+         );
+
+         const warning = screen.queryByTestId('product-warning');
+         const note = screen.queryByTestId('product-note');
+         const disclaimer = screen.queryByTestId('product-disclaimer');
+
+         (expect(warning) as any).not.toBeInTheDocument();
+         (expect(note) as any).not.toBeInTheDocument();
+         (expect(disclaimer) as any).not.toBeInTheDocument();
+      });
    });
 
-   test('descriptors do not render', async () => {
-      const productVariant = getMockProductVariant({
-         note: '',
-         warning: '',
-         disclaimer: '',
+   describe('Product Price Tests', () => {
+      test('Regular Price Renders', async () => {
+         const nonDiscountedProduct = getNonDiscountedProduct(29.99);
+
+         renderWithAppContext(
+            <ProductSection
+               product={nonDiscountedProduct}
+               selectedVariant={nonDiscountedProduct.variants[0]}
+               onVariantChange={jest.fn()}
+               internationalBuyBox={null}
+            />
+         );
+
+         const price = await screen.findByTestId('product-price');
+
+         (expect(price) as any).toBeInTheDocument();
+         (expect(price.textContent) as any).toBe('$29.99');
       });
 
-      renderWithAppContext(
-         <ProductSection
-            product={getMockProduct()}
-            selectedVariant={productVariant}
-            onVariantChange={jest.fn()}
-            internationalBuyBox={null}
-         />
-      );
-
-      const warning = screen.queryByTestId('product-warning');
-      const note = screen.queryByTestId('product-note');
-      const disclaimer = screen.queryByTestId('product-disclaimer');
-
-      (expect(warning) as any).not.toBeInTheDocument();
-      (expect(note) as any).not.toBeInTheDocument();
-      (expect(disclaimer) as any).not.toBeInTheDocument();
+      test.only('Discounted Price Renders', async () => {
```

Comment:
@ardelato I just noticed this `test.only`. Is there a reason why we're doing this?

Reply:
Ahh shoot! That was left over from testing only this test 😓

Reply:
I'll open a follow-up pull to address this. Nice catch @batbattur

Reply:
https://github.com/iFixit/react-commerce/pull/1150